build/SharedFx.targets

```diff
@@ -528,6 +528,21 @@
       Overwrite="true"/>
   </Target>
 
-  <Target Name="BuildSharedFx" DependsOnTargets="GeneratePropsFiles;ResolveSharedFxFiles;CrossGenAssemblies;CrossGenSymbols;PackSharedFx"/>
+  <Target Name="BuildSharedFx" DependsOnTargets="GeneratePropsFiles;ResolveSharedFxFiles;CrossGenAssemblies;CrossGenSymbols;PackSharedFx;TestSharedFx"/>
+
+  <Target Name="TestSharedFx" DependsOnTargets="DefineSharedFxPrerequisites;ResolveCommitHash">
+    <PropertyGroup>
+      <UnitTestFxProject>$(MSBuildThisFileDirectory)..\test\SharedFx.UnitTests\SharedFx.UnitTests.csproj</UnitTestFxProject>
```

Comment:
nit: $(RepositoryRoot) instead of $(MSBuildThisFileDirectory)..

Reply:
Updated.